.gitignore

```diff
@@ -16,3 +16,4 @@ schema.graphql
 subgraph.yaml
 build/
 *.scss.d.ts
+*.ps1
```

Comment:
what is ps1 ?

Reply:
powershell scripts

Reply:
for what it is needed/use?

Reply:
to prevent powerscript scripts from being added to the repository

Reply:
what is power scripts ? and why alchemy is using that ? did we had that before ?

Reply:
has nothing to do with Alchemy per se.  I have occasion to use powerscript scripts.

Reply:
so please exclude that from .gitignore

Reply:
.chrome and .vscode are no different in this respect, but they are very desirable.  I'll go ahead and remove .ps1 anyway since it is pretty easy to work-around.  However, it does increase the possibility of people committing files we don't want.